tests/unit/test_artifactory_path.py

```diff
@@ -751,6 +737,129 @@ def test_auth(self):
         a = P("http://a/artifactory/", auth=("foo", "bar"))
         self.assertEqual(a.auth, ("foo", "bar"))
 
+    @responses.activate
+    def test_deploy_file(self):
+        """
+        Test that file uploads to the path
+        :return:
+        """
+        P = self.cls
+        path = P(
+            "http://artifactory.local/artifactory/libs-release-local",
+            auth=("foo", "bar"),
+        )
+
+        test_file = pathlib.Path("test.zip")
```

Comment:
Let's use tmp file solution from pytest https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
@allburov 
we use UnitTests in tests. Would you like to use here https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryFile ?

Reply:
UDP2:
I was wrong, could be done with NamedTemporaryFile in a readable way. Changes pushed